src/rgw/rgw_op.cc

```diff
@@ -1036,7 +1036,7 @@ int handle_cloudtier_obj(req_state* s, const DoutPrefixProvider *dpp, rgw::sal::
       op_ret = get_system_versioning_params(s, &epoch, NULL);
       ldpp_dout(dpp, 20) << "getting versioning params tier placement handle cloud tier" << op_ret << dendl;
       if (op_ret < 0) {
-	ldpp_dout(dpp, 20) << "failed to get versioning params, op_ret = " << op_ret << dendl;
+        ldpp_dout(dpp, 20) << "failed to get versioning params, op_ret = " << op_ret << dendl;
```

Comment:
you can set default value for `flags` to true during  the definition like done for write_meta  (https://github.com/ceph/ceph/blob/main/src/rgw/driver/rados/rgw_rados.h#L833) .. that way you needn't update all the callers.